src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
 public class Duke {
+
+    private static Scanner in = new Scanner(System.in);
+    private static final Task[] tasks = new Task[100];
+
+    public static void command(String line) {
```

Comment:
A better naming practices for methods is to start it with a verb. For example, you can call this `executeCommand` or something better here. (While the word command can be a verb too, it doesn't suit your usage very well)